azure-migrate/dependencies-at-scale/AzMig_Dependencies.ps1

```diff
@@ -0,0 +1,585 @@
+
+		Write-Host "Downloading machines for appliance " $appliancename ". This can take 1-2 minutes..."
+        $batchSize = 100
+        $skipResult = 0
+        [System.Collections.Generic.List[string]]$kqlResult
+        
+        while ($true) {
+        
+          if ($skipResult -gt 0) {
+            $graphResult = Search-AzGraph -Query $query1 -First $batchSize -SkipToken $graphResult.SkipToken
+          }
+          else {
+            $graphResult = Search-AzGraph -Query $query1 -First $batchSize
+          }
+        
+          $kqlResult += $graphResult.data
+        
+          if ($graphResult.data.Count -lt $batchSize) {
+            break;
+          }
+          $skipResult += $skipResult + $batchSize
+        }
+
+        if ($kqlResult) {
+             $appliancename = $item.Key
+                    Write-Host "Machines discovered for $appliancename"
+                    $kqlResult | Export-Csv -NoTypeInformation -Path $OutputCsvFile -Append
+                    Write-Host "List of machines saved to" $OutputCsvFile
+        } else {
+            Write-Host "No results found."
+        }
+            
+            }
+        
+            
+        } 
+
+
+
+
+
+function Set-AzMigDependencyMappingAgentless {
+    [CmdletBinding()]
+    Param(
+        [Parameter(ParameterSetName = 'Enable', Mandatory = $true)]
+        [Parameter(ParameterSetName = 'Disable', Mandatory = $true)]
+        [string]$InputCsvFile,
+
+        [Parameter(ParameterSetName = 'Enable', Mandatory = $true)]
+        [switch]$Enable,
+
+        [Parameter(ParameterSetName = 'Disable', Mandatory = $true)]
+        [switch]$Disable
+    )
+
+    if (-not (Test-path -Path $InputCsvFile)) {
+        throw "File $InputCsvFile not found"
+    }
+    
+	if (-not ($InputCsvFile -imatch ".*\.csv$")) {
+        throw "Input file is not CSV."    
+    }
+	
+    
+	$VMDetails = Import-CSV $InputCsvFile
+	
+	if(-not ($VMDetails[0].psobject.Properties.Name.ToLower().contains("armid")) ) {
+		throw "Input CSV file does not contain required column 'ARMID'"
+	}
+
+    if($Enable)
+    { 
+        $ActionVerb = "Enabled";
+		$EnableDependencyMapping = $true;
+    } elseif ($Disable) {
```

Comment:
Same in these `else` statements. Previous `else` statements in the code had `else` on the next line instead of the same line. Pick a formatting standard and be consistent.